definitions/ext-firewall/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+# Firewall devices from Kentik using Metric API
+# Note this initial definition is limited to only Fortigate Firewalls using the FORTINET-FORTIGATE-MIB
+# Future firewall entities will need to be isolated in the definition
+type: FIREWALL
+synthesis:
+  name: device_name
+  identifier: device_name
+  encodeIdentifierInGUID: true
+
+  conditions:
+  - attribute: provider
+    value: kentik-firewall
+
+  tags:
+    src_addr:
+      entityTagName: device_ip
```

Comment:
what is this value? who is calling the firewall?

what is the expected amount of values here? Because I would be a bit wary of having all the internet ip's in this tag

Reply:
this is the IP of the firewall itself; should be fairly static and in almost all cases would never change